core-web/apps/dotcms-ui/src/app/portlets/dot-experiments/dot-experiments-list/components/dot-experiments-status-filter/dot-experiments-status-filter.component.spec.ts

```diff
@@ -0,0 +1,53 @@
+import { DotExperimentsStatusFilterComponent } from './dot-experiments-status-filter.component';
+import { createComponentFactory, mockProvider, Spectator } from '@ngneat/spectator';
+import {
+    DotExperimentStatusList,
+    ExperimentsStatusList
+} from '@portlets/dot-experiments/shared/models/dot-experiments.model';
+import { MultiSelect, MultiSelectModule } from 'primeng/multiselect';
+import { DotMessageService } from '@services/dot-message/dot-messages.service';
+import { Pipe, PipeTransform } from '@angular/core';
+import { FormsModule } from '@angular/forms';
+
+@Pipe({ name: 'dm' })
+class MockDmPipe implements PipeTransform {
+    transform(value: string): string {
+        return value;
+    }
+}
+
+const selectOptions = ExperimentsStatusList;
+fdescribe('DotExperimentsStatusFilterComponent', () => {
```

Comment:
remove `f`

Reply:
I will find a rule of lint to check that.